### PR TITLE
[12] Use compose profiles for toggling services; make more services optional

### PR DESCRIPTION
**Pull request by davidlougheed**
* Make Beacon optional
* Make Gohan optional

### CODE REVIEW

File: py_bentoctl/config.py [hunk 1]
```diff
@@ -61,62 +79,95 @@ def _env_get_bool(var: str, default: bool = False) -> bool:
...
+        profiles: Optional[List[str]] = v.get("profiles")
+        if profiles is None:
+            # If no profiles, always enabled
+            yield k
+            continue
+        for p in profiles:
+            if (f := BENTO_OPTIONAL_FEATURES_BY_PROFILE.get(p)) is not None and f.enabled:
+                yield k
+                break
+            if p == BENTO_PROFILE_DEV:
```
Comment (v-rocheleau) [inline py_bentoctl/config.py:148]: Dont see any "dev" profiles in the compose yamls, what is the expected behaviour here?

**Review by v-rocheleau — COMMENTED** [review]
Looks good from a quick look, good idea to bundle features like that. Will try locally soon. These changes dont have incidence on how the command line tool is used right?

File: py_bentoctl/config.py (hunk 1 above)
Comment (brouillette) [reply py_bentoctl/config.py:148]: i think we maintain the status quo of separating prod/dev by docker-compose files, and only use profiles to toggle services

Comment (davidlougheed) [reply py_bentoctl/config.py]: removed

**Comment by davidlougheed** [comment]
yes cli should be the same. i've addressed the comment made.

**Review by v-rocheleau — APPROVED** [review]
lgtm

**Pull request merged**